tests/ui/abi/compatibility.rs

```diff
@@ -1,16 +1,159 @@
 // check-pass
+// revisions: host
+// revisions: arm
+//[arm] compile-flags: --target arm-unknown-linux-gnueabi
+// revisions: aarch64
+//[aarch64] compile-flags: --target aarch64-unknown-linux-gnu
+// revisions: s390x
+//[s390x] compile-flags: --target s390x-unknown-linux-gnu
+// revisions: mips
+//[mips] compile-flags: --target mips-unknown-linux-gnu
+// revisions: mips64
+//[mips64] compile-flags: --target mips64-unknown-linux-gnuabi64
+// revisions: sparc
+//[sparc] compile-flags: --target sparc-unknown-linux-gnu
+// revisions: sparc64
+//[sparc64] compile-flags: --target sparc64-unknown-linux-gnu
+// revisions: powerpc64
+//[powerpc64] compile-flags: --target powerpc64-unknown-linux-gnu
+// revisions: riscv
+//[riscv] compile-flags: --target riscv64gc-unknown-linux-gnu
+// revisions: loongarch64
+//[loongarch64] compile-flags: --target loongarch64-unknown-linux-gnu
+// revisions: wasm
+//[wasm] compile-flags: --target wasm32-unknown-unknown
```

Comment:
Maybe add an extra revision for wasm32-wasi?

Reply:
All right, done.